app/(moderation)/layout.tsx

```diff
@@ -9,6 +12,7 @@ export default function DashboardLayout({
 }) {
   return (
     <>
+      <RejectingReason />
```

Comment:
**nitpick:** Currently we don't know how and where this modal is being called. We should place it where we call it.